### PR TITLE
[No MRG] build matio as external project

### PR DESCRIPTION
**Pull request by massich**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -29,10 +29,10 @@ Cite this software
 
 The references to be acknowledged are ::
 
-    Gramfort et al. OpenMEEG: opensource software for quasistatic
+    Gramfort A, Papadopoulo T, Olivi E, Clerc M. OpenMEEG: opensource software for quasistatic
     bioelectromagnetics. Biomedical engineering online (2010) vol. 9 (1) pp. 45
 
-    Kybic et al. Generalized head models for MEG/EEG: boundary element method
+    Kybic J, Clerc M, Abboud T, Faugeras O, Keriven R, Papadopoulo T. Generalized head models for MEG/EEG: boundary element method
```
Comment (agramfort) [inline README.rst]: @massich why is this here?

Comment (massich) [reply README.rst]: 'cos I messed up triggering travis. It should not be here. since it is already in master. So I don't really understand it.

**Comment by agramfort** [comment]
it's because you like very much rewriting git history :)

**Comment by massich** [comment]
closed in favor of #299
